ui/StatusQ/src/StatusQ/Components/StatusMessageHeader.qml

```diff
@@ -61,82 +59,168 @@ Item {
                     root.clicked(this, mouse)
                 }
             }
-        }       
-        StatusBaseText {
-            id: messageOriginInfo
-            verticalAlignment: Text.AlignVCenter
-            visible: text
-            color: Theme.palette.baseColor1
-            font.pixelSize: Theme.asideTextFontSize
-            text: root.messageOriginInfo
         }
-        StatusContactVerificationIcons {
-            id: verificationIcons
-            visible: !root.amISender
-            isContact: root.isContact
-            trustIndicator: root.trustIndicator
+
+        Loader {
+            id: messageOriginInfoLoader
+            sourceComponent: messageOriginInfoComponent
```

Comment:
Since we only don't switch the `sourceComponent` and the Components below are only used once in their respective Loaders, would you mind putting the Component directly inside the Loader. I feel it's easier to understand and creates a little less clutter.